pynicotine/gtkgui/frame.py

```diff
@@ -939,16 +955,19 @@ def add_preferences_item(self, menu):
         menu.setup(("#" + _("_Preferences"), "app.settings"))
 
     def add_quit_item(self, menu):
-        menu.setup(("#" + _("_Quit"), "app.quit"))
+
+        ellipsis = "…" if config.sections["ui"]["exitdialog"] else ""
+
+        menu.setup(
+            ("", None),
+            ("#" + _("_Quit") + ellipsis, "app.quit")
```

Comment:
You already defined a translated string with the ellipsis included in trayicon.py, use it instead.

Reply:
Do tray menus support mnemonics? I can't test right now.

Reply:
No, they do not.

Reply:
You've already added a new string anyway, so might as well finish the work properly here. Otherwise we have to change it later if RTL languages are added.